app/controllers/api/project.php

```diff
@@ -161,18 +161,16 @@
         } catch (DuplicateException $th) {
             throw new Exception(Exception::VARIABLE_ALREADY_EXISTS);
         }
-        $dbForConsole->deleteCachedDocument('projects', $project->getId());
 
         $functions = $dbForProject->find('functions', [
             Query::limit(APP_LIMIT_SUBQUERY)
         ]);
 
         foreach ($functions as $function) {
             $dbForProject->updateDocument('functions', $function->getId(), $function->setAttribute('live', false));
+            $dbForProject->deleteCachedDocument('functions', $function->getId());
```

Comment:
This isn't needed because the updateDocument() should purge the cache, right?

https://github.com/utopia-php/database/blob/cc0247f4f0c402b39f663bf9f77b29d69b95f9d6/src/Database/Database.php#L2928

Reply:
True, removed from multiple places (everywhere we set live=false)